apps/docs/src/app/platform/component-docs/platform-feed-input/platform-feed-input-docs.component.html

```diff
@@ -0,0 +1,52 @@
+<h2>Standard</h2>
```

Comment:
You should use `fd-docs-section-title` component for documentation header.

Reply:
@salarenko Changed